visualstimulation/analysis.py

```diff
@@ -36,7 +37,7 @@ def compute_dsi(rates, orients):
     out : float
         direction selectivity index
     """
-    from visualstimulation.helper import wrap_angle, find_nearest
+    from .helper import wrap_angle, find_nearest
```

Comment:
Use absolute imports instead, e.g. visualstimulation.utils.  Absolute imports are more portable and usually more readable. Also relative imports are highly discouraged by PEP8.